code/code/misc/combat.cc

```diff
@@ -3976,7 +3976,7 @@ int TBeing::oneHit(TBeing *vict, primaryTypeT isprimary, TThing *weapon, int mod
      if(doesKnowSkill(SKILL_CHAIN_ATTACK) &&   // must know the skill
 	 !weapon &&                             // must be barehanded
 	 (::number(0,99) < 10) &&                // only 10% of the time
-	 getMana()>=10 &&                       // requires 10 mana
+	 getMana()>=5 &&                       // requires 5 mana
```

Comment:
I haven't used blur much but I feel like the chain attack mana cost is fine. I was planning to allow chain attack to work with weapons as part of my "make multi classing more fun" changes. If you want to update the cost though you'll need to update the reconcileMana() call below to match this.

Reply:
Oops. Thanks!

Monks tend to blow through mana REALLY fast. 
Blur doubles their attacks in a given round, which for many is 4-5 x 2. Whereas chain attack just gives 1 more hit. The mana effectiveness is not balanced there imo.

I also feel that in prolonged combat, or even in plow situations, monks run out of mana entirely too quickly and essentially lose their spam powers in the first minute or so. If this is too OP we can roll it back.

If you plan on making weapons work for monks I suggest we should finish the 'Martial Weapon' object type code and create proper martial arts weapons to work this way.